.github/workflows/ci.yaml

```diff
@@ -29,31 +29,17 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 20
+          node-version: 23
```

Comment:
I suggest this uses an LTS version? (i.e. 22)

Reply:
Saw this comment, not sure if still applies:

> I left the integration tests in CJS so that the behaviour of requiring yargs in a CJS environment can be tested -- this requires Node 23.

Reply:
LTS version is working now. When I started this work the LTS version wasn't yet working for requiring ESM in CJS.